rssfeed/models.py

```diff
@@ -4,10 +4,11 @@
 class Feed(models.Model):
     title = models.CharField(max_length=2000, blank=True, null=True)
```

Comment:
According to the spec for RSS 2.0, there is no limit for character data that can be contained in an RSS element, so since most databases handle charfield as max 255, maybe its best to limit them to 255?